src/Entity/OgRole.php

```diff
@@ -6,177 +6,206 @@
  */
 namespace Drupal\og\Entity;
 
-use Drupal\Core\Entity\ContentEntityBase;
-use Drupal\Core\Entity\EntityTypeInterface;
-use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Config\Entity\ConfigEntityBase;
 
 /**
- * todo: Find a way to attach the roles to group:
- *  - UUID?
- *  - Service?
- *
- * Convert into
- *
- *
- * @ContentEntityType(
+ * @ConfigEntityType(
```

Comment:
This has a lot of the same changes at https://github.com/amitaibu/og/pull/77 in, which one will we be going for with this entity stuff? :)